post-install.yml

```diff
@@ -0,0 +1,146 @@
+      register: dashboard_git_remote
+
+# deploy from localhost (api server doesn't have firewall open to push to gandalf)
+    - name: Git pull dashboard
+      git:
+        repo: https://github.com/tsuru/tsuru-dashboard.git
+        dest: /tmp/tsuru-dashboard
+      delegate_to: "localhost"
+    - name: Git push dashboard to gandalf
+      shell: >
+        git push {{ dashboard_git_remote.stdout }} master
+      args: 
+        chdir: /tmp/tsuru-dashboard
+      delegate_to: "localhost"
+
+    - name: Tsuru app-info dashboard
+      shell: >
+        tsuru app-info -a dashboard
+      register: dashboard_app_info
+
+# This deploys postgresapi, but to fully operate more work needs to be done 
+# outside this repository (firewall configuration, ssl certificates etc.)
+    - name: Tsuru app-create postgresapi
+      ignore_errors: yes
+      shell: >
+        tsuru app-create postgresapi python
+      when: "not 'postgresapi' in tsuru_apps.stdout"
+    - name: Get gandalf repo for postgresapi
+      shell: >
+        tsuru app-info -a postgresapi | grep -o 'git@.\+'
+      register: postgresapi_git_remote
+
+    - name: Git pull postgresapi
+      git:
+        repo: https://github.com/tsuru/postgres-api
+        dest: /tmp/postgresapi
+      delegate_to: "localhost"
+    - name: Git push postgresapi to gandalf
+      shell: >
+        git push {{ postgresapi_git_remote.stdout }} master
+      args:
+        chdir: /tmp/postgresapi
+      register: postgresapi_push_result
+      delegate_to: "localhost"
+
+    - name: Tsuru app-info postgresapi
+      shell: >
+        tsuru app-info -a postgresapi
+      register: postgresapi_app_info
+
+    - name: Tsuru set environment variables for postgresapi
+      shell: >
+        tsuru env-set -a postgresapi POSTGRESAPI_DATABASE=postgresapi;
+        tsuru env-set -a postgresapi POSTGRESAPI_USER=postgresapi;
+        tsuru env-set -a postgresapi POSTGRESAPI_PASSWORD={{ pg_apiuser_pass }};
+        tsuru env-set -a postgresapi POSTGRESAPI_HOST={{ postgres_host }};
+        tsuru env-set -a postgresapi POSTGRESAPI_PORT=5432;
+        tsuru env-set -a postgresapi POSTGRESAPI_SALT=md5;
+        tsuru env-set -a postgresapi POSTGRESAPI_SHARED_HOST={{ postgres_host }};
+        tsuru env-set -a postgresapi POSTGRESAPI_SHARED_PORT=5432;
+        tsuru env-set -a postgresapi POSTGRESAPI_SHARED_ADMIN=postgresadmin;
+        tsuru env-set -a postgresapi POSTGRESAPI_SHARED_ADMIN_PASSWORD={{ pg_admin_pass }};
+        tsuru env-set -a postgresapi POSTGRESAPI_SHARED_PUBLIC_HOST={{ postgres_host }};
+      when: "not 'Everything up-to-date' in postgresapi_push_result.stderr"
+    - name: Tsuru app-run db upgrade
+      shell: >
+        tsuru app-run --app postgresapi -- python manage.py upgrade_db
+      when: "not 'Everything up-to-date' in postgresapi_push_result.stderr"
+
+    - debug: var=dashboard_app_info.stdout_lines
+    - debug: var=postgresapi_app_info.stdout_lines
```

Comment:
Yes - it's useful to have the dashboard and postgresapi URLs shown at the end of the run.